packages/notifier/README.md

```diff
@@ -1,59 +1,127 @@
-# NotifierKits and SubscriptionKits
+# PublishKit and Related Types
```

Comment:
I think the primary use of this doc will be for someone what wants to understand the recommended way to use the package. The deprecated aspects are for understanding code you encounter but won't write.

As such I suggest having the README clearly document non-deprecated PublishKit (and NotifierKit?) but only mention the deprecated ones. Expect someone to understand non-deprecated before learning about them. Then have a separate document for understanding the older types, including how they work and why they were deprecated.

Reply:
I think this is currently the place to document all three, and am trying to minimize disruption to just what is necessary for including publish kits (especially since https://github.com/endojs/endo/issues/1035 will prompt further changes).